CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
 NetFoundry welcomes all and any contributions. All open source projects managed by NetFoundry share a common
-[guide for contributions](https://openziti.github.io/policies/CONTRIBUTING.html).
+[guide for contributions](https://openziti.github.io/docs/introduction/openziti-is-software/#contributing).

```

Comment:
this link needs to exist and this should be undone: https://openziti.github.io/policies/CONTRIBUTING.html we just need to make sure that old links is there. this is the worng link. i never caught that before

Reply:
Which one is the wrong link? https://openziti.github.io/policies/CONTRIBUTING.html is preserved by Docusaurus redirects and it can stay that way. This diff updates the contribution guide to point to the real page (the target of the redirect) instead of the old redirect.

Reply:
@dovholuknf clarifying question for you in this conversation

Reply:
https://openziti.github.io/policies/CONTRIBUTING.html  is mapping over to https://openziti.github.io/docs/introduction/openziti-is-software/ which is "the wrong place".  The old docfx site had https://github.com/openziti/ziti-doc/blob/pre-docusaurus/docfx_project/policies/CODE_OF_CONDUCT.html

and https://github.com/openziti/ziti-doc/blob/pre-docusaurus/docfx_project/policies/CONTRIBUTING.html

we need to resurrect the `/policies` subfolder

Reply:
peek at https://github.com/openziti/ziti-doc/tree/pre-docusaurus/docfx_project/policies in general. we lost /policies in the docfx --> docusaurus change

Reply:
OK, self-assigned https://github.com/openziti/ziti-doc/issues/228 and will look into lost policies and update the redirect.

Reply:
reverted change to URL